gr 3.10.6.0 fails to build with libfmt 10

### What happened?

Build fails on all arches

### System Information

Fedora Linux

### GNU Radio Version

3.10 (maint-3.10)

### Specific Version

3.10.6.0

### Steps to Reproduce the Problem

Attempt to build with libfmt 10.

### Relevant log output

```shell
/builddir/build/BUILD/gnuradio-3.10.6.0/gr-uhd/lib/rfnoc_rx_streamer_impl.cc:144:23:   required from here
/usr/include/fmt/core.h:1691:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1691 |       formattable,
      |       ^~~~~~~~~~~
/usr/include/fmt/core.h:1691:7: note: 'formattable' evaluates to false
```
https://kojipkgs.fedoraproject.org//work/tasks/8847/102698847/build.log
```

--- a/gr-uhd/lib/rfnoc_rx_streamer_impl.cc
+++ b/gr-uhd/lib/rfnoc_rx_streamer_impl.cc
@@ -143,7 +143,7 @@ int rfnoc_rx_streamer_impl::work(int noutput_items,
     default:
         d_logger->warn("RFNoC Streamer block received error {:s} (Code: {})",
                        d_metadata.strerror(),
-                       d_metadata.error_code);
+                       static_cast<int>(d_metadata.error_code));
     }
 
     if (d_metadata.end_of_burst) {

